File: lua/test/exists_test.lua
-- LmEmail SDK exists test

local sdk = require("lm-email_sdk")

describe("LmEmailSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)

File: rb/test/exists_test.rb
# LmEmail SDK exists test

require "minitest/autorun"
require_relative "../LmEmail_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = LmEmailSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
